src/utils/mavenUtils.ts

```diff
@@ -38,7 +38,7 @@ export namespace mavenUtils {
         });
     }
 
-    export async function executeMvnCommand(telemetryProperties: TelemetryProperties, outputChannel: vscode.OutputChannel | undefined, workingDirectory: string | undefined, ...args: string[]): Promise<string> {
+    export async function executeMvnCommand(telemetryProperties: TelemetryProperties | undefined, outputChannel: vscode.OutputChannel | undefined, workingDirectory: string | undefined, ...args: string[]): Promise<string> {
```

Comment:
Why did you change this function? When is telemetryProperties undefined?

Reply:
This is due to this method signature:
```javascript
getTemplates(language: string, runtime: string = ProjectRuntime.one, functionAppPath: string, templateFilter?: string, telemetryProperties?: TelemetryProperties)
```
If the `telemetryProperties` is not optional, we need to pass it in the unit test